### PR TITLE
Fix HARNESS_DIR path for db-extras-zynq7

### PR DESCRIPTION
**Pull request by tmichalak**
This commit fixes issue #1050 and most probably #1049 as well.

### CODE REVIEW

**Comment by tmichalak** [comment]
@mithro 
I didn't find anything in the failing fuzzer. It passed locally even when when I downloaded the failed fuzzer that is packed on error by kokoro.
I rerun the CI and it passed so maybe this was some kind of a glitch?

File: Makefile [hunk 1]
```diff
@@ -137,7 +137,7 @@ db-extras-zynq7:
 	+source minitests/roi_harness/zybo-swbut.sh && $(MAKE) -C fuzzers part_only
 	+source minitests/roi_harness/zybo-swbut.sh && \
 		$(MAKE) -C minitests/roi_harness \
-			HARNESS_DIR=database/artix7/harness/zybo/swbut run
```
Comment (mithro) [inline Makefile]: Thus should probably be `run copy`?

Comment (tmichalak) [reply Makefile]: Well, I was about to ask about that.
The minitests/roi_harness/Makefile is:
<pre>
copy: $(HARNESS_FILES)
    mkdir -p $(HARNESS_DIR)
    cp $(HARNESS_FILES) $(HARNESS_DIR)

run:
    make clean
    make copy
</pre>
So why do we additionally explicitly call ``run copy`` in artix7's db-extras?

Comment (mithro) [reply Makefile]: Dunno...

Comment (tmichalak) [reply Makefile]: @litghost I think you added the ``run copy`` to db-extras-artix7`` (correct me if I am wrong). Maybe we should remove the redundant ``copy`` there and this one is good to merge.

**Comment by tmichalak** [comment]
@litghost @mithro Do you insist on adding the `run copy` instead of simply having `run`?
If not then there is no other reason keeping this small PR from being merged.

**Pull request merged**